app/views/reviews/_list_reviews.html.erb

```diff
@@ -1,11 +1,10 @@
-<div class="col-lg-8 col-md-12 border-item post-list">
-  <div class="comments-area">
-    <h1><%= t("views.accounts.review.reviews", total: @account.reviews.count) %></h1>
-    <div class="comment-list">
-      <% @account.reviews.each do |review| %>
-        <%= render partial: "reviews/review", locals: {review: review} %>
-      <% end %>
-    </div>
+<div class="comments-area">
+  <h1><%= t("views.accounts.review.reviews", total: reviewable.reviews.count) %></h1>
+  <div class="comment-list">
+    <% @reviews.each do |review| %>
```

Comment:
Cái này có thể viết tắt dùng render conllection được mà e
Sao phải for each như nayf?